api/envoy/config/cluster/redis/redis_cluster.proto

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
nit: del empty  beginning/trailing lines (same elsewhere in PR)

Reply:
ok